include/swift/AST/FeatureAvailability.def

```diff
@@ -76,7 +76,6 @@ FEATURE(InitRawStructMetadata,                          (6, 0))
 
 FEATURE(LayoutStringValueWitnesses,                     (6, 1))
 FEATURE(CreateTaskWithConsumedFunction,                 (6, 1))
-FEATURE(IsolatedDeinit,                                 (6, 1))
```

Comment:
Do we still need this for checking availability if the experimental feature is enabled?

Reply:
Oh I see, yeah let me bring that back.